### ISSUE TITLE
Unable to play songs with viewer-discretion warning

### ISSUE THREAD

**Issue by subby2006** [vfsfitvnm/ViMusic#1222]
### Steps to reproduce the bug

1. Search for YOASOBI's Into the Night
2. Play the video version
3. ViMusic is stuck loading the song then shows 'An unknown playback error has occured'

### Expected behavior

The song should play

### Actual behavior

'An unknown playback error has occured'

### Screenshots/Screen recordings

![Screenshot_20231013-171232](https://github.com/vfsfitvnm/ViMusic/assets/65925208/40173515-09f5-4ca8-a11b-5211d524fa03)

### ViMusic version

0.5.4

### Android version

Android 13

### Additional information

The song is flagged by YouTube and has a viewer discretion warning
![Screenshot_20231013-171321](https://github.com/vfsfitvnm/ViMusic/assets/65925208/2d4e47f7-5acf-40e9-aa5e-0e8683b9bd3c)

**Comment by zyqunix** [comment]
yes, because the app works on youtube music and there's no way to press "confirm" on it, the creator is using the API i think
